dts/bindings/base/zephyr,memory-region.yaml

```diff
@@ -16,3 +16,16 @@ properties:
       memory region in the final executable. The region address and size
       is taken from the <reg> property, while the name is the value of
       this property.
+
+  zephyr,memory-region-mpu:
+    type: string
+    required: false
+    enum:
+      - "REGION_RAM_ATTR"
```

Comment:
Thinking about it a little more, is it possible to reduce these options to `"RAM"`, `"RAM_NOCACHE"` and `"FLASH"` by adding the appropriate pre/postfix in macros? The following seems cleaner to me
```
   sram_no_cache: memory@20300000 {
        zephyr,memory-region = "SRAM_NO_CACHE";
        zephyr,memory-region-mpu = "RAM_NOCACHE";
   };
```

Reply:
yup. Done.